packages/stores/types.ts

```diff
@@ -16,6 +16,19 @@ type TAccount = NonNullable<Authorize['account_list']>[0] & {
     balance?: number;
 };
 
+type TCtraderAccountsList = {
```

Comment:
@heorhi-deriv Everything is optional? 🤔

Reply:
@farzin-deriv checking ...
@farzin-deriv found one property that is not optional 👌